package.json

```diff
@@ -44,6 +44,8 @@
     "rangy": "^1.3.0",
     "rich-text": "^3.0.1",
     "sharedb": "^1.0.0-beta",
+    "sharedb-mongo": "^1.0.0-beta.6",
```

Comment:
Added these two with npm install so that the build would work again. I'm not sure if all the changes added to the package-lock are good or required, but it built after I made these changes.